FeedStoreChallenge/CoreDataModels/ManagedCache.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Husam Dayya on 17/03/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedCache)
+public class ManagedCache: NSManagedObject {
+    @NSManaged public var timestamp: Date
+    @NSManaged public var feed: NSOrderedSet
+	
+	@nonobjc public class func fetchRequest() -> NSFetchRequest<ManagedCache> {
+		return NSFetchRequest<ManagedCache>(entityName: ManagedCache.entity().name!)
+	}
+}
```

Comment:
This class and all its properties and methods shouldn't be public. They're internal implementation details.

```suggestion
class ManagedCache: NSManagedObject {
    @NSManaged var timestamp: Date
    @NSManaged var feed: NSOrderedSet
	
	@nonobjc class func fetchRequest() -> NSFetchRequest<ManagedCache> {
		return NSFetchRequest<ManagedCache>(entityName: ManagedCache.entity().name!)
	}
}
```